src/components/shared/index.js

```diff
@@ -0,0 +1,22 @@
+import QuanthubMessage from '@/components/QuanthubMessage.vue'
+import QuanthubCloseIcon from '@/components/shared/icons/QuanthubCloseIcon.vue'
+import QuanthubModal from '@/components/shared/QuanthubModal.vue'
+import BadgeLabel from '@/components/shared/BadgeLabel.vue'
+import CustomButton from '@/components/shared/CustomButton.vue'
+import CustomSelect from '@/components/shared/CustomSelect.vue'
+
+const registerBootstrapComponents = (vue) => {
```

Comment:
Not sure this is correctly named as bootstrap has gone? Also it would be good to have consistent naming. e.g. why badge-label and custom-button. Do they all need to be quanthub?

Reply:
Renamed.